client/src/api/index.js

```diff
@@ -1,4 +1,4 @@
-const API_ROOT_URL = '/api'
+const API_ROOT_URL = ''
```

Comment:
I'm guessing this is temporary until we decide what we want to do with it for the API refactor.

Reply:
exactly!